masonry/src/passes/event.rs

```diff
@@ -165,7 +165,7 @@ pub fn root_on_text_event(
     handled
 }
 
-pub fn root_on_access_event(
+pub(crate) fn root_on_access_event(
```

Comment:
(Had commented directly on the commit, but then that doesn't show up here ...)

I use this in my own event loop integration. I need some way to pump an access event into the system ...

Reply:
My mistake. I'm calling this on `RenderRoot`, not this one here.